src/sources/syslog.rs

```diff
@@ -355,7 +359,17 @@ pub fn udp(
             let host_key = host_key.clone();
             async move {
                 match frame {
-                    Ok(((mut events, _byte_size), received_from)) => {
+                    Ok(((mut events, byte_size), received_from)) => {
+                        emit!(SocketBytesReceived {
+                            mode: SocketMode::Udp,
+                            byte_size,
+                        });
+                        let count = events.len();
+                        emit!(SocketEventsReceived {
```

Comment:
Not sure if this is being emitted in the right place. I'll double check what the tcp implementation does later on my second pass on this PR once the other comments are addressed